docs/source/how-to/docker.md

```diff
@@ -0,0 +1,44 @@
+# Run Tiled using Docker
+
+To run tiled using docker we must first obtain a docker image containing a tiled installation.
+To do this run 
+
+```
+docker pull ghcr.io/bluesky/tiled:main
+```
+
+In this example we will use the docker image to serve a directory of files as explained [here](../../tutorials/serving-files.md).
+
+First generate a directory of example files using a utility provided by Tiled.
+
+```
+python -m tiled.examples.generate_files example_files/
+```
+
+The docker container runs tiled via [gunicorn](https://gunicorn.org/) which provides horizontal scaling over workers.
+Because of this we cannot configure tiled using commandline arguments and must specify a config file.
+
+Take the following server configuration below:
+```yaml
+# config.yml
+trees:
+  - path: /
+    tree: tiled.trees.files:Tree.from_directory
+    args:
+      directory: "example_files"
+uvicorn:
+  host: 0.0.0.0
+  port: 8000
+authentication:
+  allow_anonymous_access: true
+```
+and serve it using the docker container
+```
```

Comment:
Do we want to use the more concise `--volume` flag rather than `--mount`?


docker run --rm -p 8000:8000 \
  --volume "/deploy/example_files: $(pwd)/config.yml" \
  --volume "$(pwd)/config.yml:/deploy/config.yml" \
  --env TILED_CONFIG=/deploy/config.yml tiled

Reply:
The docker documentation encourages the more verbose form.
https://docs.docker.com/storage/bind-mounts/
I can change it though if you prefer.

Reply:
so it does! their logic is funny "research has shown that it is easier to use"... anyway, no, don't change it.